src/yadism/coefficient_functions/light/nlo/g4.py

```diff
@@ -0,0 +1,26 @@
+import numba as nb
+import numpy as np
+from eko.constants import CF, TR
+
+from . import f2
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def ns_reg(z, _args):
```

Comment:
this piece should be the same as `f2.ns_reg`

Reply:
Where do you get that from? I got this expression from eq.17 here: https://arxiv.org/pdf/hep-ph/9412255.pdf, which also seems to be what is implemented in apfel.

Reply:
Looking [here](https://arxiv.org/pdf/2210.12014.pdf), you can see that eq 19 holds up to NNLO. 
I tried with Mathematica and the expressions are indeed the same.

Reply:
Indeed also [here](https://arxiv.org/pdf/hep-ph/9509236.pdf) seems to agree with those relations. it's also the same as what is currently implemented but I can indeed directly import the function from f2.